src/components/MultiHopTrade/hooks/useReceiveAddress.tsx

```diff
@@ -95,11 +95,14 @@ export const useReceiveAddress = ({
               pubKey: shouldFetchUnchainedAddress ? fromAccountId(buyAccountId).account : undefined,
             })
 
-            return walletReceiveAddress
+            return walletReceiveAddress ?? null
           }
         : skipToken,
     staleTime: Infinity,
   })
 
-  return { walletReceiveAddress, isLoading: isInitializing || isLoading }
+  return {
+    walletReceiveAddress: walletReceiveAddress ?? undefined,
```

Comment:
We _still_ want `useReceiveAddress` to return `undefined` here (as all of our consumers still expect this), we just don't want the query function above to do so.